vaccine/application.py

```diff
@@ -43,30 +118,213 @@ async def check_age(content: Optional[str]):
                 age = int(content or "")
                 assert age >= 0
             except (ValueError, TypeError, AssertionError):
-                raise ErrorMessage(
-                    "Sorry, we don't understand your response. "
-                    "Please type the number that represents your age in years"
-                )
+                raise ErrorMessage("⚠️  Reply using numbers only. Example *27*")
 
         return FreeText(
             self,
-            question="What is your current age, in years?",
-            next="state_end",
+            question="\n".join(
+                ["◼️◼️◼️◻️◻️", "", "How old are you? (Reply with a number)"]
+            ),
+            next="state_location",
             check=check_age,
         )
 
-    async def state_end(self):
+    async def state_location(self):
+        return FreeText(
+            self,
+            question="\n".join(
+                [
+                    "◼️◼️◼️◼️◻️",
+                    "",
+                    "Where do you live? (Reply with name of suburb, town, settlement "
```

Comment:
Will we not need to use Maps API to look up the location like we do for Health Check?

Reply:
Yeah we can, will check with Neville because it didn't mention this in the sheet (it also doesn't mention location pins)

Reply:
Have amended the copy in the source spreadsheet as follows:

Reply:
◼️◼️◼️◼️◻️

Please tell us where you live.

You can share your location 📍 using WhatsApp by following these steps:
- Tap the attach icon (+ or  📎) on the bottom of WhatsApp. 
- Select location
- This will bring up a map with a pin showing your current location. Tap *Send Your Current Location* if this is your address.
- You can also choose from various other nearby locations if you think that the GPS is a bit off.

Or TYPE the name of your Suburb, Township, Town or Village (or nearest)?